### PR TITLE
move env_thermo precip arrays to aux

### PR DESCRIPTION
**Pull request by trontrytel**
Peel off from #506 - moves environment precipitation related data structures to aux. 

(does not affect RiCO)

### CODE REVIEW

**Review by charleskawczynski — APPROVED** [review]

File: src/EDMF_Environment.jl [hunk 1]
```diff
@@ -52,7 +54,14 @@ function sgs_mean(en_thermo::EnvironmentThermodynamics, grid, state, en, precip,
...
         )
         update_sat_unsat(en_thermo, state, k, ts)
-        update_env_precip_tendencies(en_thermo, state, k, mph.qt_tendency, mph.θ_liq_ice_tendency)
+        update_env_precip_tendencies(
+            state,
+            k,
+            mph.qt_tendency,
+            mph.θ_liq_ice_tendency,
+            mph.qr_tendency,
+            mph.qs_tendency,
```
Comment (charleskawczynski) [inline src/EDMF_Environment.jl:63]: Maybe in a later PR we can change this function to accept `mph`, that way we don't accidentally mix up the arg order.

Comment (charleskawczynski) [reply src/EDMF_Environment.jl:63]: Ahh, but one of the calls doesn't create an `mph` struct. This is fine.

Comment (trontrytel) [reply src/EDMF_Environment.jl:63]: Right. I actually started coding it that way yesterday and then reminded myself that it is also used with quadrature outputs as arguments

**Comment by charleskawczynski** [comment]
Ahh, sorry! I thought these were touching two different parts of the code base! 😬

**Comment by charleskawczynski** [comment]
Ahhh, it was the rain -> precip rename that did it, sorry!

**Comment by trontrytel** [comment]
np - will resolve now

**Pull request merged**